registry/api/v2/names.go

```diff
@@ -15,18 +15,10 @@ const (
 	// single repository name slash-delimited component
 	RepositoryNameComponentMinLength = 2
 
-	// RepositoryNameComponentMaxLength is the maximum number of characters in a
-	// single repository name slash-delimited component
-	RepositoryNameComponentMaxLength = 30
-
 	// RepositoryNameMinComponents is the minimum number of slash-delimited
 	// components that a repository name must have
 	RepositoryNameMinComponents = 1
 
-	// RepositoryNameMaxComponents is the maximum number of slash-delimited
-	// components that a repository name must have
-	RepositoryNameMaxComponents = 5
```

Comment:
Sorry, forgot to make that change. I knew that was there but with
everything going on today I missed it. I'll get it in later tonight.
On Fri, Mar 6, 2015 at 6:03 PM Stephen Day <notifications@github.com> wrote:

> In registry/api/v2/names.go
> <https://github.com/docker/distribution/pull/242#discussion_r25985918>:
>
> >  	// RepositoryNameMinComponents is the minimum number of slash-delimited
> >  	// components that a repository name must have
> >  	RepositoryNameMinComponents = 1
> >
> > -	// RepositoryNameMaxComponents is the maximum number of slash-delimited
> > -	// components that a repository name must have
> > -	RepositoryNameMaxComponents = 5
>
> RepositoryNameRegexp needs to be modified to not restrict this, as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/distribution/pull/242/files#r25985918>.
>